blueocean-config/src/main/resources/io/jenkins/blueocean/config/BlueOceanConfig/header.jelly

```diff
@@ -0,0 +1,18 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+  <script>
+    (function () {
+      window.$$blueoceanConfig = JSON.parse('${it.blueOceanConfig}');
```

Comment:
Seems like this plugin is no longer just a dogfood thing, right? If so, then maybe we should (at a min) store this inside the `jenkinsCIGlobal` object, alongside other Jenkins related stuff (where `js-modules` stores stuff).

In reality though, I'd really like to discourage all but `js-modules` from storing things in the global scope (and for it to just store refs to commonjs modules). We could add this in the page markup and have a javascript module for reading it. And to the js-extensions stuff in the same way (kill 2 birds).